base_station/gui/src/components/DriveVelDataH.vue

```diff
@@ -0,0 +1,223 @@
+      <td class = "tableElement">3</td>
+      <td class = "tableElement">{{drive_vel_data.middleRight.current}} amps</td>
+      <td class = "tableElement">{{drive_vel_data.middleRight.velocity*60}} rpm</td>
+      <td class = "tableElement">5</td>
+      <td class = "tableElement">{{drive_vel_data.backRight.current}} amps</td>
+      <td class = "tableElement">{{drive_vel_data.backRight.velocity*60}} rpm</td>
+    </tr>
+  </tbody>
+  </table>
+  <div>
+        <h3> Drive State </h3>
+  </div>
+  <table class="tableFormat" style="undefined;table-layout: fixed; width: 255px">
+      <colgroup>
+        <col style="width: 85px">
+        <col style="width: 85px">
+        <col style="width: 85px">
+      </colgroup>
+      <thead>
+          <tr class="bold">
+              <td class = "tableElement">Front</td>
+              <td class = "tableElement">Middle</td>
+              <td class = "tableElement">Back</td>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td class = "tableElement">{{drive_state_data.leftState}}</td>
+              <td class = "tableElement">{{drive_state_data.rightState}}</td>
+              <td class = "tableElement">{{drive_state_data.rightState}}</td>
+          </tr>
+      </tbody>
+  </table>
+</div>
+</template>
+
+<style scoped>
+    .wrap {
+        display: inline-block;
+        align-content: center;
+        /* height: 300px; */
+    }
+    .box {
+        border-radius: 5px;
+        padding: 10px;
+        border: 1px solid black;
+        text-align: right;
+        vertical-align: top;
+    }
+    .tableFormat{
+    border-collapse:collapse;
+    border-spacing:0;
+    }
+    .tableFormat td{
+    border-color:black;
+    border-style:solid;
+    border-width:1px;
+    font-size:13px;
+    overflow:hidden;
+    padding:10px 5px;
+    word-break:normal
+    }
+    .tableFormat th{
+    border-color:black;
+    border-style:solid;
+    border-width:1px;
+    font-size:13px;
+    font-weight:normal;
+    overflow:hidden;
+    padding:10px 5px;
+    word-break:normal;
+    }
+    .bold{
+      font-weight: bold;
+      border: 2px solid black;
+    }
+    .tableFormat .tableElement{
+    border-color:inherit;
+    text-align:center;
+    vertical-align:top
+    }
+</style>
+
+<script>
+export default {
+    data() {
+      return {
+        drive_vel_data: {
+          frontLeft:{current: 0, velocity: 0},
+          frontRight:{current: 0, velocity: 0},
+          middleLeft:{current: 0, velocity: 0},
+          middleRight:{current: 0, velocity: 0},
+          backLeft:{current: 0, velocity: 0},
+          backRight:{current: 0, velocity: 0}
+        },
+        drive_state_data: {
+          leftState: "Boot",
+          rightState: "Boot"
+        }
+      }
+    },
+    created: function(){
+      this.$parent.subscribe('/drive_vel_data', (msg) => {
+        if(msg.axis == 0){
+          this.drive_vel_data.frontLeft.current = msg.measuredCurrent;

```

Comment:
This stuff won't work since the DriveVelData lcm changed from measuredCurrent and estimatedVel to current_amps and vel_percent